lib/src/main/kotlin/at/bitfire/ical4android/AndroidEvent.kt

```diff
@@ -104,6 +77,14 @@ abstract class AndroidEvent(
          * The URL is directly put into [ExtendedProperties.VALUE].
          */
         const val MIMETYPE_URL = ContentResolver.CURSOR_ITEM_BASE_TYPE + "/vnd.ical4android.url"
+
+        /**
+         * Google Calendar uses an extended property called `iCalUid` for storing the event's UID, instead of the
+         * standard [Events.UID_2445].
+         *
+         * @see <a href="https://github.com/bitfireAT/ical4android/issues/125">GitHub Issue</a>
+         */
+        const val GCALENDAR_ICAL_UID = "iCalUid"
```

Comment:
We should decide how to name the constants that contain `ExtendedProperties.NAME`s and then apply the same scheme to all properties – otherwise confusion may occur at some point.

I originally used the `MIMETYPE_` prefix because I think they should be formatted as MIME types (found that somewhere in the docs). However in reality they're often just a string (like `iCalUid`).

So we should either name everything `MIMETYPE_`, then it would be `MIMETYPE_ICAL_UID`.

Or (maybe better) we should prefix everything (including `CATEGORIES` and `URL`) with `EXTNAME_`. But in any case all things of the same type should have the same prefix (and defined alphabetically so that a certain property can be found quickly).

Reply:
I prefer the `EXTNAME_` syntax better than `MIMETYPE_`.